src/osemgrep/core/Profiler.mli

```diff
@@ -0,0 +1,29 @@
+(** A really basic and linear profiler.
```

Comment:
what does linear mean in this context?

Reply:
Maybe you can add a comment like

related: libs/profiling/Profiling.ml also provide a poor's man profiler, but here we want to restrict profiling
to broad categories that are compatible with pysemgrep, so better to define another module for it.